includes/admin/discounts/discount-actions.php

```diff
@@ -76,6 +76,7 @@ function edd_add_discount( $data ) {
 
 		if ( edd_store_discount( $posted ) ) {
 
+			wp_cache_flush();
```

Comment:
This flushes the entire cache, not just the one for discounts. Could we avoid that?

Reply:
Nope the only cache deletion available requires a hash and since the hashes are based on args the only way to guarantee it's  a cleared is to kill it all  There is not a way to delete an entire group. We _could_ look at supporting that non-persistent flag, I linked you to yesterday.

Reply:
Then  how about replacing the hash with a predictable cache key? Deleting _all_ caches is just being a bad citizen.